INSTALL.md

```diff
@@ -1,11 +1,16 @@
 Installing From Sources
 =======================
 
-This document presents instructions to install this branch of Coq.
-For more general installation instructions and information about known
-build system issues, please consult the wiki page: 
+To install and use Coq, we recommend relying on [the Coq
+platform](https://github.com/coq/platform/) or on a package manager
+(e.g. opam or Nix).
 
-  https://github.com/coq/coq/wiki#coq-installation
+See https://coq.inria.fr/download and
+https://github.com/coq/coq/wiki#coq-installation to learn more.
+
+If you really need to build Coq from sources manually (e.g. to
+contribute to Coq or to write a Coq package), then the remainder of this
+file explains how to.
```

Comment:
```suggestion
file explains how to do this.
```
Sounded incomplete to my ear.

Reply:
Yup.  Or could just be "... explains how."